content/projects/gsoc/mentors.adoc

```diff
@@ -47,18 +45,281 @@ Mentors are expected to...
+
+Then we send our slot request min and max numbers to Google.
+
+==== During project selection phase
+
+*This phase lasts one week.*
+
+This phase start immediately when Google sends us our final number of slots. We may get only the minimum.
+Sometimes heart wrenching decisions need to be made at this time.
+
+During this phase, mentors and Jenkins Org Admins hold a private meeting to make the final project selection
+and the mentor teams are finalized and confirmed. Then we submit our final selection to the GSoC program.
+
+=== During the final selection awaiting period
+
+*This phase lasts a few days.*
+
+We wait. We are NOT allowed to communicate any information regarding the selection to students.
+
+=== During Community Bonding
+
+*This phase lasts 3 weeks.*
+
+*Expectation:* about 5 to 8 hours per week.
+
+This is the most critical phase when it comes to working with your student.
+Year after year, if this phase goes well, the rest of the program usually goes well, but if this phase does not go well, the project usually fails.
+
+Do not waste any time.
+Organize your first official meeting with the student, within the first week. Meet every week immediately.
+We have written a guide for this phase, link:../students/#community-bonding[read it] and follow it.
+
+In rare cases, it is acceptable to re-work the project idea, even change it completely, as long as the student and the mentors all agree.
+Written documentation about this is essential, if it happens, let the Org Admins know.
+
+Let Jenkins Org Admins know as soon as possible if there is any need for knowledge transfer sessions.
+We often organize special public presentations to go over plugin and core development flows and code that
+students and mentors alike benefit from attending.
+
+=== During Coding Phases
+
+*Expectation:* about 5 to 8 hours per week.
+
+See also: link:../students/#coding-periods[student coding periods].
+
+You are mentoring now. Guide, coach, review pull-requests, unblock the student,
+ensure students use Jira for task, bug and feature tracking,
+meet your student at least once or twice per week over a live one-on-one session
+(video conferencing and screen sharing is infinitely useful here).
+
+What if the student is done early? Student and mentor must determine other work that can be done and let the Jenkins Org Admins know.
+
+=== During evaluation periods
+
+*Expectation:* same as during the coding periods
+
+See also: link:../students/#evaluations[student evaluations].
+
+Mentors are expected to evaluate their student, while they continue to mentor them.
+Coding seldom completely stops during this period.
+
+The evaluation is done by comparing the student project plan with the actual code produced.
+There is usually great flexibility as we allow mentor and student to agree on what the expectation are in terms of features and content.
+
+In the Jenkins project, we ask that students prepare slides using Google Slides,
```

Comment:
Maybe mentioning a specific tech is an overkill

Reply:
Removed specific technology mention.